### DISCUSSION TITLE
Run scripts on a schedule and system checks

### DISCUSSION THREAD

**Discussion by MrVinyla** [Ylianst/MeshCentral#4570] [Ideas]
It would be great if MeshCentral had the ability to run scripts on a schedule, as well as the ability to perform automatic system checks at set times. Thank you

**Comment by dinger1986** [comment]
Something like tactical rmm?

**Reply by MrVinyla** [reply]
I don't know about tactical rmm, but like SolarWinds MSP RMM

**Reply by dinger1986** [reply]
Yeah tactical rmm is a source available rmm. Tasks/schedules I believe are being worked on in meshcentral

**Reply by Miner-Ninja** [reply]
> SolarWinds MSP RMM

It would be a really powerful feature of Meshcentral to have the ability to run scripts on a schedule

**Reply by silversword411** [reply]
Currently MeshCentral supports plugins with scripting: https://ylianst.github.io/MeshCentral/meshcentral/plugins/ . I don't think there's any scheduling support.

If you want more, use an RMM like https://github.com/amidaware/tacticalrmm

**Reply by Miner-Ninja** [reply]
> Currently MeshCentral supports plugins with scripting: https://ylianst.github.io/MeshCentral/meshcentral/plugins/ . I don't think there's any scheduling support.
> 
> If you want more, use an RMM like https://github.com/amidaware/tacticalrmm

Wow wow ...
Thank you for info
I will try this plugin ;)
